src/wp-admin/css/common.css

```
@@ -1289,11 +1300,13 @@ th.action-links {
 }
 
 .filtered-by .tags {
-	display: inline;
+        display: flex;
+        align-items: flex-start;
+        flex-wrap: wrap;
+	gap: 8px;
```

```suggestion
	display: flex;
	align-items: flex-start;
	flex-wrap: wrap;
	gap: 8px;
```

The first three lines still indent with spaces (probably because I edited the `gap` line in two separate comments).